src/js/feedback/components/FeedbackForm.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AlertBox from '../../common/components/AlertBox';
+
+const MIN_INPUT = 1;
+
+class FeedbackForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
I think we've generally leaned toward using Redux for state (https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Engineering/Style%20Guides/Javascript/vets.gov%20Front%20End.md#using-setstate-in-react-components), unless it's pretty emphemeral UI state. Why'd you decide to make this local state vs putting it with the other Redux state?

Reply:
Yeah I was keeping the form values in local state until the form is submitted, then it makes it into Redux.

Reply:
I don't think there's ever a clear answer to that question, but I think we've tried to keep things in Redux unless we have a reason not to.

Reply:
I guess I usually look at in inversely, in that I try to keep object state as contained as possible and only let it propagate throughout Redux state if there's a clear incentive to do so.

Reply:
I don't think that's the wrong approach necessarily, but you lose one of the benefits of Redux, which is that all (or the vast majority) of your state is in one place and is changed in similar ways.

In any case: we don't have a hard and fast rule about this and I don't think this way is wrong, just that our other apps more often take a different approach.